.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
```

I'm not sure exactly why CI isn't running, but maybe this will do it:
```suggestion
  pull_request:
    branches: [master]
```

---

Huh, that didn't seem to do it. Maybe Actions doesn't run workflows added through forks by default and there's something else that has to be enabled?